torch/autograd/gradcheck.py

```diff
@@ -140,8 +132,8 @@ def _get_numerical_jacobian(fn, inputs, outputs=None, target=None, eps=1e-3,
     if target is None:
         target = inputs
     inp_indices = [i for i, a in enumerate(target) if is_tensor_like(a) and a.requires_grad]
-    for i, (inp, inp_idx) in enumerate(zip(iter_tensors(target, True), inp_indices)):
-        jacobians += [get_numerical_jacobian_wrt_specific_input(fn, inp, inp_idx, inputs, outputs, eps, grad_out)]
+    for inp_idx in inp_indices:
+        jacobians += [get_numerical_jacobian_wrt_specific_input(fn, inp_idx, inputs, outputs, eps, grad_out)]
```

Comment:
it seemed cleaner to just pass in the inputs and the indices of the inputs (that are tensors and require grad)